docs/getting-started.md

```diff
@@ -356,9 +347,6 @@ module.exports = {
             options: {
               // Prefer Dart Sass
               implementation: require('sass'),
-
-              // See https://github.com/webpack-contrib/sass-loader/issues/804
-              webpackImporter: false,
```

Comment:
nit: Also update [Appendix section](https://github.com/material-components/material-components-web/blob/1ca94e7e01a4b345d0a91a504811de61c4a78959/docs/getting-started.md#appendix-configuring-a-sass-importer-for-nested-node_modules) to remove webpackImporter config.

Reply:
Thanks!